ukpa/templates/ukpa/nomination.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'ironcage/base.html' %}
+{% load avatar_tags %}
+
+{% block content %}
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+  {% avatar user %}<br>
+    <h4>{{ form.description.label }}</h4>
+    <div>{{ nomination.statement|linebreaks }}</div>
+    <hr />
+
+    <a href="{% url 'ukpa:nomination_delete' nomination.nomination_id %}" class="btn btn-danger">Withdraw your nomination</a>
```

Comment:
This is because the `delete_nomination` view only accepts `POST`s, and clicking a link always sends a `GET`.

It'd also be good to have an "are you sure" popup -- could we use jQuery to render this popup when the link is clicked, and then to submit a `POST`?